docs/useGetOne.md

```diff
@@ -90,7 +90,7 @@ const UserProfile = () => {
 };
 ```
 
-This results in less calls to the dataProvider. For instance, if the `<UserProfile>` component above is rendered in a `<Datagrid>`, it will only make one call to `dataProvider.getMany()` for the entire list instead of one call to `dataProvider.getOne()` per row.
+This results in less calls to the dataProvider. For instance, if the `<UserProfile>` component above is rendered in a `<DataTable>`, it will only make one call to `dataProvider.getMany()` for the entire list instead of one call to `dataProvider.getOne()` per row.
```

Comment:
```suggestion
This results in less calls to the dataProvider. For instance, if the `<UserProfile>` component above is rendered in a [`<DataTable>`](./DataTable.md], it will only make one call to `dataProvider.getMany()` for the entire list instead of one call to `dataProvider.getOne()` per row.
```